new_lineage_panel/src/components/Column.tsx

```diff
@@ -0,0 +1,111 @@
+import { FunctionComponent, ReactElement, useMemo } from "react";
+import BooleanIcon from "../assets/icons/boolean.svg?react";
```

Comment:
you could follow [this pattern](https://github.com/AltimateAI/vscode-dbt-power-user/blob/master/webview_panels/src/assets/icons/index.ts#L1) to avoid multiple imports